frontend/api/index.js

```diff
@@ -136,7 +130,7 @@ if (process.env.FLAGSMITH_PROXY_API_URL) {
 if (isDev) {
   // Serve files from src directory and use webpack-dev-server
   // eslint-disable-next-line
-    console.log('Enabled Webpack Hot Reloading');
+  console.log('Enabled Webpack Hot Reloading')
```

Comment:
The indentation changes here are good, but I'm not sure we should be removing the semi colon?

Reply:
Looks like pre-commit's prettier hook ignores the `eslint-disable-next-line` annotation. Reverted.